src/Features/LanguageServer/Protocol/Handler/References/FindAllReferencesHandler.cs

```diff
@@ -22,12 +23,16 @@ namespace Microsoft.CodeAnalysis.LanguageServer.Handler
     internal class FindAllReferencesHandler : AbstractStatelessRequestHandler<LSP.ReferenceParams, LSP.ReferenceItem[]?>
     {
         private readonly IMetadataAsSourceFileService _metadataAsSourceFileService;
+        private readonly IAsynchronousOperationListener _asyncListener;
 
         [ImportingConstructor]
         [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
-        public FindAllReferencesHandler(IMetadataAsSourceFileService metadataAsSourceFileService)
+        public FindAllReferencesHandler(
+            IMetadataAsSourceFileService metadataAsSourceFileService,
+            IAsynchronousOperationListenerProvider asynchronousOperationListenerProvider)
         {
             _metadataAsSourceFileService = metadataAsSourceFileService;
+            _asyncListener = asynchronousOperationListenerProvider.GetListener(FeatureAttribute.LanguageServer);
```

Comment:
wasn't quite certain the right listener here.  it feels like it could either be .LanguageServer or .FindReferences